README.md

```diff
@@ -3,20 +3,92 @@
 GitHub Action for upload code scanning results to [Pixeebot](https://pixee.ai/)
 so it can fix the issues they found.
 
-## For Sonar Users
+## For SonarCloud Users
 
-For Sonar integration, the `pixee/upload-tool-results-action` must be configured
-to execute when the Sonar GitHub App completes a check. The `sonar-pixeebot.yml`
-example workflow includes the requisite configuration and is generic enough to
-apply to most repositories without modification.
+For SonarCloud integration, the `pixee/upload-tool-results-action` must be
+configured to execute when the SonarCloud GitHub App completes a check. The
+`sonar-pixeebot.yml` example workflow includes the requisite configuration and
+is generic enough to apply to most repositories without modification.
 
-1. Copy the [example sonar-pixeebot.yml](./examples/sonar-pixeebot.yml) workflow
-   to the repository's `.github/workflows` directory.
+1. Copy the
+   [example sonarcloud-pixeebot.yml](./examples/sonarcloud-pixeebot.yml)
+   workflow to the repository's `.github/workflows` directory.
 1. Set the `SONAR_TOKEN` secret. Create a SonarCloud token at
    [https://sonarcloud.io/account/security](https://sonarcloud.io/account/security).
    See
    [Using secrets in GitHub Actions](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions).
 
+## For SonarQube Users
+
+For SonarQube integration, the `pixee/upload-tool-results-action` must be
+configured to execute after the SonarQube has completed its analysis. In a
+typical GitHub Action workflow that includes SonarQube, the step that performs
+the SonarQube analysis will be followed by a step that applies the SonarQube
+Quality Gate; the `pixee/upload-tool-results-action` should follow the SonarQube
+Quality Gate. The workflow should be configured to run the
+`pixee/upload-tool-results-action` step, regardless of the outcome of the
+quality gate, so that Pixeebot may fix the issues preventing the quality gate
+from passing.
+
+The `pixee/upload-tool-results-action` requires a SonarQube _user token_ token
+that is permitted to read Security Hotspots. Recommend providing this token as a
+new secret `PIXEE_SONAR_TOKEN`. Typically, the `SONAR_TOKEN` secret in a GitHub
```

Comment:
```suggestion
new secret `PIXEE_SONAR_TOKEN` in order to prevent conflict with any preexisting `SONAR_TOKEN`. Typically, the `SONAR_TOKEN` secret in a GitHub
```